globalization/overview.md

```diff
@@ -14,42 +14,106 @@ Internationalization (I18N) is the process of making an app support different la
-
-* [DateInput](https://demos.telerik.com/blazor-ui/dateinput/globalization) - @[template](/_contentTemplates/common/general-info.md#date-format-per-culture)
-
-* [DatePicker](https://demos.telerik.com/blazor-ui/datepicker/globalization) - @[template](/_contentTemplates/common/general-info.md#date-format-per-culture)
-
-* [DateRangePicker](https://demos.telerik.com/blazor-ui/daterangepicker/globalization) - @[template](/_contentTemplates/common/general-info.md#date-format-per-culture)
-
-* [DateTimePicker](https://demos.telerik.com/blazor-ui/datetimepicker/globalization) - @[template](/_contentTemplates/common/general-info.md#date-format-per-culture)
-
-* [Grid](https://demos.telerik.com/blazor-ui/grid/globalization) - the various inputs and editors are Telerik components and respond to the culture. Custom code and format strings in the templates will also default to using the current culture unless you explicitly use a certain culture in them.
-
-* [NumericTextBox](https://demos.telerik.com/blazor-ui/numerictextbox/globalization) - the `Format` (for example, currency), decimal separator, group separator and default number of `Decimals` are taken from the current culture.
-
-* [RangeSlider](https://demos.telerik.com/blazor-ui/rangeslider/globalization) - The labels on the large ticks are formatted according to the current culture and its default decimals.
-
-* [Scheduler](https://demos.telerik.com/blazor-ui/scheduler/globalization) - Date formats are taken from the culture, in the week view the first day of the week is also taken from the culture even if it does not match the `Date`. The various inputs and editors in the edit form are Telerik components and respond to the culture.
-
-* [Slider](https://demos.telerik.com/blazor-ui/slider/globalization) - The labels on the large ticks are formatted according to the current culture and its default decimals.
-
-* [TimePicker](https://demos.telerik.com/blazor-ui/timepicker/globalization) - @[template](/_contentTemplates/common/general-info.md#date-format-per-culture)
-
-* [TreeList](https://demos.telerik.com/blazor-ui/treelist/globalization) - the various inputs and editors are Telerik components and respond to the culture. Custom code and format strings in the templates will also default to using the current culture unless you explicitly use a certain culture in them.
+* [Globalization (G11N)]({%slug globalization-formats%}) - the ability to react to the chosen culture where format strings are involved (such as number and date formats).
+
+* [Right-to-Left Support]({%slug rtl-support%}) - the ability to render the components in a right-to-left direction instead of the default left-to-right direction. This feature will become available in an upcoming release.
```

Comment:
```suggestion
* [Right-to-Left Support]({%slug rtl-support%})—the ability to render the components in a right-to-left direction instead of the default left-to-right direction. This feature will become available in an upcoming release.
```